src/mina-signer/src/field-bigint.ts

```diff
@@ -155,11 +155,15 @@ function checkRange(lower: bigint, upper: bigint, name: string) {
   return (x: bigint) => {
     if (x < lower)
       throw Error(
-        `${name}: inputs smaller than ${lower} are not allowed, got ${x}`
+        `${name}: inputs smaller than ${lower} are not allowed, got ${x} \
+        ${name === 'Scalar' ? 'You may want to use `convertPrivateKeyToBase58WithMod` before using this key.' : ''}
+        `
```

Comment:
nice!
could you just remove the `\` and make it so that the error message starts at the beginning of the new line?
same for the "larger" case below

```suggestion
        `${name}: inputs smaller than ${lower} are not allowed, got ${x}.
${name === 'Scalar' ? 'You may want to use `convertPrivateKeyToBase58WithMod` before using this key.' : ''}
`
```